docs/config/node.rst

```diff
@@ -205,6 +208,93 @@ Ports
   range is used. If this is set to an integer value it is considered as an
   explicit single port.
 
+Advanced TCP settings
+---------------------
+
+Any interface that uses TCP (PG/HTTP/transport) shares the following settings:
+
+.. _network.tcp.no_delay:
+
+**network.tcp.no_delay**
+  | *Default:* ``true``
+  | *Runtime:* ``no``
+
+  Enable or disable the `Nagle's algorithm`_ for buffering TCP packets.
+  Buffering is disabled by default.
+
+.. _network.tcp.keep_alive:
+
+**network.tcp.keep_alive**
+  | *Default:* ``true``
+  | *Runtime:* ``no``
+
+  Configures the ``SO_KEEPALIVE`` option for sockets, which determines
+  whether they send TCP keepalive probes.
+
+.. _network.tcp.reuse_address:
+
+**network.tcp.reuse_address**
+  | *Default:* ``true on non-windows machines and false otherwise``
+  | *Runtime:* ``no``
+
+   Configures the `` SO_REUSEADDRS`` option for sockets, which determines
+   whether they should reuse the address.
+
+.. _network.tcp.send_buffer_size:
+
+**network.tcp.send_buffer_size**
+  | *Default:* ``-1``
+  | *Runtime:* ``no``
+
+  The size of the TCP send buffer. By default not explicitly set.
+
+.. _network.tcp.receive_buffer_size:
+
+**network.tcp.receive_buffer_size**
+  | *Default:* ``-1``
+  | *Runtime:* ``no``
+
+  The size of the TCP receive buffer. By default not explicitly set.
+
+.. NOTE::
+
+    Each setting in this section has its counterpart for HTTP and transport.
+    To provide a protocol specific setting, remove ``network`` prefix and use
+    either ``http`` or ``transport`` instead. For example, no_delay can be
+    configured as ``http.tcp.no_delay`` and ``transport.tcp.no_delay``. Please
+    note, that PG interface takes its settings from transport.
+
+Transport settings
+------------------
+
+.. _transport.connect_timeout:
+
+**transport.connect_timeout**
+  | *Default:* ``30s``
+  | *Runtime:* ``no``
+
+  The connect timeout for initiating a new connection.
+
+.. _transport.compress:
+
+**transport.compress**
+  | *Default:* ``false``
+  | *Runtime:* ``no``
+
+  Set to `true` to enable compression (DEFLATE) between all nodes.
+
+.. _transport.ping_schedule:
+
+**transport.ping_schedule**
+  | *Default:* ``-1``
+  | *Runtime:* ``no``
+
+  Schedule a regular application-level ping message to ensure that transport
+  connections between nodes are kept alive. Defaults to `-1` (disabled). It is
```

Comment:
in the referenced version default is either 5s or -1 (transport only) but our version has only -1 default